### DISCUSSION TITLE
Download directory help

### DISCUSSION THREAD

**Discussion by SteamRex25** [aclap-dev/video-downloadhelper#1284] [Help and Q&A]
### Your question

Hello, I am hoping to find a way to change the download directory for videos that I download as there is a specific folder that I wish to place them into but I have found no way to do so at this point

### Addon & CoApp version and details

I have the latest version of the application and the extension as I just downloaded them today. (I am on the linux version)

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

You will want to read #665 & #814.  There's a bit of reading there but you should make the time to read it all carefully.  It will answer most of your questions, I'm sure.  But if you still have more questions after you read all of that, don't hesitate to ask here.  On the other hand, if I can tell from your questions that you actually have not read all of that, I just won't answer you.

In addition to what you will read there, VDH has been improved since then & it now indeed does keep a history log of your downloads.  Hunt for the relevant icon in the VDH menu & experiment with it.  That will do you much more good than reading some documentation on it.

**Comment by SteamRex25** [comment]
so, I have figured out how to download to a specific directory but it doesn't seem to remember where I want to download it too if I don't select download as. Thanks for helping me get to this point however

**Reply by GrampaWildWilly** [reply]
I'm not sure I understand.  When you do `Download`, does it not always go into whatever you have in your VDH settings for the download directory?  Now, if you want specific control over the target directory, then yes, `Download As` is required.  Personally, I never do `Download`.  I always do `Download As`.  Every `Download As` starts by showing whichever directory I used the previous time I did a `Download As`.  I can change that, but it starts with the same one I used last time.  `Download` does not give you that choice, but it should still always download into the same directory.  You can manually change that target in the VDH settings, but that's cumbersome, tedious.  Much simpler to just always use `Download As`.

You must understand that the target directory for `Download` & the target directory for `Download As` are not connected.  They are remembered in 2 different places inside VDH.  They do not have to be the same directory.  And doing a `Download As` into a directory does NOT change the target directory for your next `Download`.  Nor vice versa.  They are just 2 separate target directories & they are not related in any way.

**Reply by GrampaWildWilly** [reply]
You probably need to read #665 & #814 again slowly & carefully.  Be sure you look at the images & read the notes within the images.  In particular, #814 shows you how to change your default action.  You probably want to change your default action to `Download As`.
